src/main/java/ch/njol/skript/events/SimpleEvents.java

```diff
@@ -509,6 +510,11 @@
 						"	set the fake max players count to (online players count + 1)",
 						"	set the shown icon to a random server icon out of {server-icons::*}")
 				.since("2.3");
+		if(Skript.classExists("org.bukkit.event.command.UnknownCommandEvent")) Skript.registerEvent("Unknown Command", SimpleEvent.class, UnknownCommandEvent.class, "unknown command")
```

Comment:
Same thing here... move ``Skript.registerEvent(...)`` onto the next line.

Reply:
Done